{"uuid": "3191632a-eb1b-4616-9ba5-91980c525e61", "children": ["3a3ecd68-2657-4fa2-953e-85bc5a6723db"], "befores": [{"name": "open_app_without_login", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"G:\\android_automation\\conftest.py\", line 56, in open_app_without_login\n    service.start()\n  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\appium\\webdriver\\appium_service.py\", line 186, in start\n    main_script: str = kwargs.get('main_script') or get_main_script(node, npm)\n  File \"C:\\Users\\a.jamshidi\\android_automation\\android_automation\\.venv\\lib\\site-packages\\appium\\webdriver\\appium_service.py\", line 81, in get_main_script\n    modules_root = sp.check_output([npm_path] + args).strip().decode('utf-8')\n  File \"C:\\Users\\a.jamshidi\\AppData\\Local\\Programs\\Python\\Python38\\lib\\subprocess.py\", line 415, in check_output\n    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,\n  File \"C:\\Users\\a.jamshidi\\AppData\\Local\\Programs\\Python\\Python38\\lib\\subprocess.py\", line 495, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"C:\\Users\\a.jamshidi\\AppData\\Local\\Programs\\Python\\Python38\\lib\\subprocess.py\", line 1015, in communicate\n    stdout = self.stdout.read()\n"}, "start": 1729603130869, "stop": 1729603133566}], "start": 1729603130869, "stop": 1729603133752}